lib/mpv/utils.rb

```diff
@@ -12,5 +12,10 @@ def self.which?(util)
         File.executable?(File.join(path, util))
       end
     end
+
+    def self.tmpname
```

Comment:
One minor nit: Let's call this `tmpsock`, since it's explicitly appending `.sock` to the end of the tempnames it creates.